javascript/javascript1/week1/pizza-exercise/part4.js

```diff
@@ -0,0 +1,32 @@
+console.log("==== PART 4====");
+
+const orderNumber2 = [1, 2, 2]; // These are Pesto, Pep og Pep without any discounts
+console.log("Order:", orderNumber2);
+
+const totalPrice2 =
+  pizzaPrices[order[1]] + pizzaPrices[order[2]] + pizzaPrices[order[2]];
+console.log("Total price:", totalPrice2.toFixed(2), "DKK");
+
+// Adding 20% off the first pizza
+const discount2 = pizzaPrices[order[0]] * 0.2;
+
+// new discounted total price
+const discountedPrice2 = totalPrice2 - discount2;
+
+// Results
+console.log("Discount of first pizza:", discount2.toFixed(2), "DKK");
+
+// Plastic bag
+
+const addPlasticBag = 5 + ".00" + " DKK";
+console.log("Added plastic bag:", addPlasticBag);
+
+const priceWithBag = discountedPrice2 + addPlasticBag;
```

Comment:
`addPlasticBag` is a string. It will mess up the math on line 24.
Instead do something like this:
```suggestion
const addPlasticBag = 5;
console.log(`Added plastic bag: ${addPlasticBag} DKK`);

const priceWithBag = discountedPrice2 + addPlasticBag;
```